src/main/java/com/google/flourbot/entity/EntityInterface.java

```diff
@@ -0,0 +1,8 @@
+package com.google.flourbot.entity;
+
+import java.lang.Exception;
+import com.google.flourbot.datastorage.FirebaseDataStorage;
+
+public interface EntityInterface {
+    public Macro getMacro(String userEmail, String macroName) throws Exception;
+}
```

Comment:
Interface for Entity Module, which calls converts Map from datastore into a Macro object.